modules/swagger-codegen/src/main/resources/typescript-inversify/apiInterface.mustache

```diff
@@ -0,0 +1,26 @@
+{{>licenseInfo}}
+import { Headers } from "../Headers";
+import { Observable } from "rxjs/Observable";
+import * as models from "../model/models";
+
+{{#operations}}
+
+{{#description}}
+    /**
+    * {{&description}}
+    */
+{{/description}}
+export interface {{classname}}Interface {
+    [others: string]: any;
```

Comment:
is this needed?

Reply:
If you set the additional parameters `withInterfaces`, the `apiInterface` should be necessary.

Reply:
I was referring to `[others: string]: any;`